.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,20 +1,24 @@
-<!-- Put a description of what this PR is for here, along with any references to issues that this resolves or contributes to -->
+## Description
 
+_Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request.
+If it fixes a bug or resolves a feature request, be sure to link to that issue._
 
 
-<!-- AUTOCHANGELOG for Downstream PRs.
 
-Please select one of the following "release-note:" headings:
-    - release-note:enhancement
-    - release-note:bug
-    - release-note:feature
-    - release-note:change
-    - release-note:none
+## Type of change
```

Comment:
Would it make sense to add examples of good release notes? In kubernetes/kubernetes, contributors have access to the following [document](https://github.com/kubernetes/community/blob/master/contributors/guide/release-notes.md#applying-a-release-note) with release note examples. I think this could improve the overall quality of the changelogs.

Reply:
Good point! I think we can potentially align with k/k recommendations and link to the same document. In addition it would be good to check what tooling they are using to generate release notes from the PRs as it may solve our problem too.

Reply:
Added few examples in PR template comment. After some investigation into how release notes are gathered in k/k I think it isn't a good fit for a smaller repo like prometheus-operator. Especially note categorization (feature, enhancement, etc.) is the thing that would be problematic if we adopted the same scheme as k/k.